version.txt

```diff
@@ -1 +1 @@
-1.33.0-beta.0
\ No newline at end of file
+1.33.0-rc.0
```

Comment:
can you skip this as we don't cherry pick the version changes as update to this would try to create a new releases.